now/utils.py

```diff
@@ -125,9 +125,7 @@ def jina_auth_login():
 
 
 def get_info_hubble(user_input):
-    with open(user('~/.jina/config.json')) as fp:
-        config_val = json.load(fp)
-        user_token = config_val['auth_token']
+    user_token = hubble.utils.auth.Auth.get_auth_token()
     client = hubble.Client(token=user_token, max_retries=None, jsonify=True)
```

Comment:
```suggestion
    client = hubble.Client(max_retries=None, jsonify=True)
```
Just omit the token parameter will be enough.

Reply:
Thanks for pointing that out! I didn't know that, I will make this change.

Reply:
To get the token `Auth.get_auth_token()`.
Or `client._token`.